Tasks/PublishSymbolsV2/task.json

```diff
@@ -13,7 +13,7 @@
     "preview": false,
     "version": {
         "Major": 2,
-        "Minor": 198,
+        "Minor": 199,
         "Patch": 0
     },
     "minimumAgentVersion": "1.95.0",
```

Comment:
Since we are adding Node10 execution, the min agent version should be 2.144.0, as per - https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode10.md .
And if we are changing the min agent version, I would suggest to use the latest.